examples/district/app/snapshot/page.tsx

```diff
@@ -0,0 +1,360 @@
+'use client';
+
+import React from 'react';
+import Image from 'next/image';
+import {
+  Box,
+  Button,
+  Divider,
+  Menu,
+  RadioGroup,
+  RadioItem,
+  Sheet,
+  Tabs,
+  Text,
+} from '@opub-cdl/ui';
+import { Carousel } from '@opub-cdl/ui/src';
+import { IconArrowLeft, IconArrowRight } from '@tabler/icons-react';
+
+import styles from './snapshot.module.scss';
+
+export default function Snapshot() {
+  const res = [
```

Comment:
you can move this and indicator list to `config/site` or some other similar file outside this component